### PR TITLE
Reshape starter course template

### PR DESCRIPTION
**Pull request by coatless**
PR refreshes the `pl-template` course further and provides a welcome splash README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
-# Template course for PrairieLearn
+# PrairieLearn Course Repository
 
-This course contains examples of PrairieLearn questions. You can see a live version at https://prairielearn.engr.illinois.edu/
+Welcome to PrairieLearn! 
+
+Within this repository, the content for your course is stored.
```
Comment (nicknytko) [inline README.md:5]:
```suggestion
The content for your course is stored within this repository.
```

I feel like this flows a bit better?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+The content for your course is stored within this repository.
+You can see and edit a live version at <https://prairielearn.engr.illinois.edu/>.
+
+### Overview
+
+PrairieLearn stores data for a course using folders and JSON files. 
+Each of these files can be edited directly using [PrairieLearn website's "Files" or "Settings" tab](https://prairielearn.engr.illinois.edu/). 
+The roles of each file and directory are:
+
+- **[infoCourse.json](infoCourse.json)**: Store basic information about the course such as its number and name, e.g. PL 101: Using PrairieLearn.
```
Comment (nicknytko) [inline README.md:14]:
```suggestion
- **[infoCourse.json](infoCourse.json)**: Stores basic information about the course such as its number and name, e.g. PL 101: Using PrairieLearn.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+
+### Welcome to PrairieLearn! 
+
+The content for your course is stored within this repository.
+You can see and edit a live version at <https://prairielearn.engr.illinois.edu/>.
+
+### Overview
+
+PrairieLearn stores data for a course using folders and JSON files. 
+Each of these files can be edited directly using [PrairieLearn website's "Files" or "Settings" tab](https://prairielearn.engr.illinois.edu/).
```
Comment (nicknytko) [inline README.md:11]:
```suggestion
Each of these files can be edited directly using the [PrairieLearn website's "Files" or "Settings" tab](https://prairielearn.engr.illinois.edu/). 
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,33 @@
...
+
+### Overview
+
+PrairieLearn stores data for a course using folders and JSON files. 
+Each of these files can be edited directly using [PrairieLearn website's "Files" or "Settings" tab](https://prairielearn.engr.illinois.edu/). 
+The roles of each file and directory are:
+
+- **[infoCourse.json](infoCourse.json)**: Store basic information about the course such as its number and name, e.g. PL 101: Using PrairieLearn.
+- **[courseInstances/](courseInstances/)**: Directory containing per-semester offerings of a course, e.g. Fall 2020 and Spring 2021.
+   - **[courseInstances/Fa20/infoCourseInstance.json](courseInstances/Fa20/infoCourseInstance.json)**: The upcoming Fall 2020 semster is found here. Make sure to set when the course should open and list the course's teaching staff.
```
Comment (nicknytko) [inline README.md:16]:
```suggestion
   - **[courseInstances/Fa20/infoCourseInstance.json](courseInstances/Fa20/infoCourseInstance.json)**: The upcoming Fall 2020 semester is found here. Make sure to set when the course should open and list the course's teaching staff.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,33 @@
...
+   - **[courseInstances/Fa20/assessments](courseInstances/Fa20/assessments)**: Any assessment such as homework or exams for the Fall 2020 semester will be located here. These assessments are specific to the single-time course offering.
+- **[questions/](questions/)**: All questions created for a course are found here. 
+   - **[questions/template](questions/template)**: Inside of the **[questions/](questions/)** directory, templates containing the best design practices for creating assessment items can be found. We recommend using these templates as a base for creating your own questions.
+
+### Reference Guide
+
```
Comment (nicknytko) [inline README.md:22]: Maybe just something along the lines of "Bookmark or save this reference guide as a cheat-sheet to use while developing questions."

Comment (coatless) [reply README.md:22]: Switched the reference guide image size to be miniaturized and opted for the pages to be side-to-side.

**Comment by nicknytko** [comment]
woops sorry, didn't know you were still hacking on this

**Comment by coatless** [comment]
@nicknytko thoughts on the README?

**Pull request merged**